flink-table/flink-sql-client/pom.xml

```diff
@@ -7,7 +7,9 @@ regarding copyright ownership.  The ASF licenses this file
 to you under the Apache License, Version 2.0 (the
 "License"); you may not use this file except in compliance
 with the License.  You may obtain a copy of the License at
+
```

Comment:
revert this

Reply:
ok, I just want to unify the format because the license header is different with other pom.xml